File: ARCADE/The Core/Level 2_8: metroCard.cs
    using System;
    using System.Collections.Generic;
     
    public class Test
    {
    	public static int[] metroCard(int lastNumberOfDays) {
    	    var myList = new List<int>();
    	    if (lastNumberOfDays == 30 || lastNumberOfDays == 28) myList.Add(31);
    	    if(lastNumberOfDays == 31) {
    	        myList.Add(28); myList.Add(30); myList.Add(31);
    	    }
    	    var a = myList.ToArray();
    	    return a;
    	}
    	public static void Main()
    	{
    		int lastNumberOfDays = 30;
    		var result = metroCard(lastNumberOfDays);
    		foreach(int i in result){
    			Console.Write(i +" ");
    		}
    	}
    }

File: ARCADE/The Core/Level 2_8: metroCard.pas
program metroCard;
Type 
    ArrayInteger = Array of integer;
    
function metroCard(lastNumberOfDays: integer): ArrayInteger;
var A : ArrayInteger;
begin
    if((lastNumberOfDays = 30) or (lastNumberOfDays = 28)) then
    begin 
        SetLength(A, 1);
        A[0] := 31;
    end
    else if(lastNumberOfDays = 31) then
    begin
        SetLength(A, 3);
        A[0] := 28;
        A[1] := 30;
        A[2] := 31;
    end;
    metroCard := A;
end;

var lastNumberOfDays, i : integer;
    A : ArrayInteger;
begin
	lastNumberOfDays := 30;
	A := metroCard(lastNumberOfDays);
	for i := 0 to length(A) - 1 do 
	    write(A[i]);
end.
